api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/MedicationRequestTranslatorImpl.java

```diff
@@ -67,6 +69,11 @@ public MedicationRequest toFhirResource(DrugOrder drugOrder) {
 		return medicationRequest;
 	}
 	
+	@Override
+	public DrugOrder toOpenmrsType(MedicationRequest medicationRequest) {
+		return toOpenmrsType(new DrugOrder(), medicationRequest);
+	}
+	
```

Comment:
I don't think we actually want this method.

Reply:
@ibacher your right though this method came about as a result to using `UpdatableOpenmrsTranslator` as explained above

Reply:
See above.